example/BouncyLayout_Example.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
```

Comment:
Maybe the gitignore should be updated to ignore these files?

Reply:
actually this one should be there. https://stackoverflow.com/questions/53034303/should-xcshareddata-be-checked-into-git-repo-for-xcode-project/53039267